### PR TITLE
ET-915 correctly detect ticket provider

### PR DESCRIPTION
**Pull request by cliffordp**
:ticket: [ET-915] correctly detect ticket provider. Artifact: https://share.getcloudapp.com/2Nuj2P9l

[ET-915]: https://moderntribe.atlassian.net/browse/ET-915

### CODE REVIEW

File: src/Tribe/Tickets.php [hunk 1]
```diff
@@ -3225,14 +3234,12 @@ public function maybe_redirect_to_attendees_registration_screen( $redirect = nul
 
 			$url = $attendee_reg->get_url();
 
-			$provider = tribe_get_request_var( 'provider' );
-
-			if ( empty( $provider ) ) {
-				$provider = $this->attendee_object;
+			if ( empty( $q_provider ) ) {
```
Comment (sc0ttkclark) [inline src/Tribe/Tickets.php:3237]: This isn't needed anymore after adding the conditional to set it above

**Review by sc0ttkclark — CHANGES_REQUESTED** [review]
Slight tweak

File: src/Tribe/Tickets.php (hunk 1 above)
Comment (cliffordp) [reply src/Tribe/Tickets.php:3237]: good catch

**Comment by cliffordp** [comment]
@sc0ttkclark ready for your review again

File: src/Tribe/Attendee_Registration/Meta.php [hunk 2]
```diff
@@ -11,16 +11,19 @@ class Tribe__Tickets__Attendee_Registration__Meta {
 	 * Add a product-deletion parameter to the shopping URL on Paypal in order to clear old products
 	 * if the order is cancelled within Paypal.
 	 *
-	 * @since 4.9
+	 * @since  4.9
 	 *
-	 * @param $args
+	 * @param array $args
```
Comment (tr1b0t) [inline src/Tribe/Attendee_Registration/Meta.php:16]: :no_entry_sign: Missing parameter comment

<sub>hash: a82fe85e81f5cfebff21c8b210b3e9ec</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 1 error(s)

**Review by sc0ttkclark — APPROVED** [review]

**Pull request merged**